assets/src/modules/Search.js

```diff
@@ -99,7 +99,6 @@ export default class Search {
                     "repository": globalThis['lizUrls'].params.repository,
                     "project": globalThis['lizUrls'].params.project,
                     "query": $('#search-query').val(),
-                    "bbox": extent.toBBOX()
                 }
```

Comment:
Ok to remove the bbox parameter.

I think we should also go further and remove the test of the intersection between the retrieved features bbox and the current extent, line 118 & 119 of the modified file